### PR TITLE
eiger-io: bump version number to 2.0.0

### PR DESCRIPTION
**Pull request by mrakitin**

### CODE REVIEW

**Comment by jrmlhermitte** [comment]
I'm not sure why, but tag v2.0.0 was behind by 2 commits of v1.0.0.
I have updated v2.0.0 to point to current `master` which is 2 commits ahead of v1.0.0.
We tested this version so it makes sense. 

However, when I build from home I get a SHA256 mistmatch:
```
Success
SHA256 mismatch: '619d221651551920b2c039e2af438412dec8a1aaf3096a759e30e0c02be671fa' != '2de8b9fae8f9d9821f3380582e1d3af0213a172fc60086f4859e6b7ff19c3f0e'
```

@licode do you have an idea why that could be?
I run `conda build -c lightsource2-tag meta.yaml` (this `meta.yaml`)

**Comment by licode** [comment]
no need to use sha. Remove sha256 line https://github.com/NSLS-II/lightsource2-recipes/pull/240/files#diff-5f5827ba037c22814bac6958a2bff747L9, and simplify this meta.yaml file as https://github.com/NSLS-II/lightsource2-recipes/blob/master/recipes-tag/bluesky/meta.yaml

**Comment by mrakitin** [comment]
Thanks @licode, update is in fbc5adc.

File: recipes-tag/eiger-io/meta.yaml [hunk 1]
```diff
@@ -1,36 +1,35 @@
...
-  name: eiger-io
-  version: {{ version }}
+    name: eiger-io
+    version: {{ version }}
 
 source:
-  url: https://github.com/NSLS-II-CHX/eiger-io/archive/v{{ version }}.tar.gz
-  fn: eiger-io-v{{ version }}.tar.gz
-  sha256: 2de8b9fae8f9d9821f3380582e1d3af0213a172fc60086f4859e6b7ff19c3f0e
+    url: https://github.com/NSLS-II-CHX/eiger-io/archive/v{{ version }}.tar.gz
```
Comment (jrmlhermitte) [inline recipes-tag/eiger-io/meta.yaml]: maybe `git-url: https://github.com/NSLS-II-CHX/eiger-io` and `git-rev: v{{ version }}
` as @licode suggested?
(https://github.com/NSLS-II/lightsource2-recipes/blob/master/recipes-tag/bluesky/meta.yaml#L8)
just to maintain consistency

**Comment by mrakitin** [comment]
Done.

**Comment by licode** [comment]
I am not sure about commits issue with different version numbers

**Comment by licode** [comment]
current one looks good to me.

**Pull request merged**